i18n/translationProvider.go

```diff
@@ -48,6 +49,18 @@ func (tp *TranslationProvider) Update(d *deps.Deps) error {
 
 	i18nBundle := bundle.New()
 
+	langs := []string{}
+	for _, currentSource := range sources {
+		for _, r := range currentSource.Files() {
+			langs = append(langs, r.BaseFileName())
+		}
+	}
+	// we need to register all language codes as "plural spec" to prevent errors
```

Comment:
Add dots at the end of sentences and please start sentences with capital lettes.